velox/functions/prestosql/tests/ProbabilityTest.cpp

```diff
@@ -155,5 +155,24 @@ TEST_F(ProbabilityTest, binomialCDF) {
       binomialCDF(-2, 0.5, -1), "numberOfTrials must be greater than 0");
 }
 
+TEST_F(ProbabilityTest, poissonCDF) {
+  const auto poissonCDF = [&](std::optional<double> lambda,
+                              std::optional<int64_t> value) {
+    return evaluateOnce<double>("poisson_cdf(c0, c1)", lambda, value);
+  };
+
+  EXPECT_EQ(0.91608205796869657, poissonCDF(3, 5));
+  EXPECT_EQ(0, poissonCDF(kDoubleMax, 3));
+  EXPECT_EQ(1, poissonCDF(3, kBigIntMax));
+  EXPECT_EQ(1, poissonCDF(kDoubleMin, 3));
+  EXPECT_THROW(poissonCDF(kInf, 3), VeloxUserError);
+  VELOX_ASSERT_THROW(
+      poissonCDF(3, kBigIntMin), "value must be a non-negative integer");
+  VELOX_ASSERT_THROW(poissonCDF(kNan, 3), "lambda must be greater than 0");
+  VELOX_ASSERT_THROW(poissonCDF(-3, 5), "lambda must be greater than 0");
```

Comment:
Move these 2 tests before all the tests with special values since they check the error cases.